### PR TITLE
Complete: Task 841-845: kde4, schema_guided_dstc8 Datasets

### PR DESCRIPTION
**Pull request by kashyap467**
Please review.

### CODE REVIEW

**Comment by swarooprm** [comment]
@Palipoor, wondering if you can help review these tasks?

File: tasks/README.md [hunk 1]
```diff
@@ -6,4 +6,9 @@ This directory contains the tasks that are part of this benchmark.
 Name | Summary | Category
 ---- | ----------- | --------
 `task1001_squad1.1_question_generation` | Generating guestions (based on SQuAD 1.1) | Question Generation  
-`task1002_squad1.1_answer_generation` | Generating answers to SQuAD 1.1 questions | Answer Generation
\ No newline at end of file
+`task1002_squad1.1_answer_generation` | Generating answers to SQuAD 1.1 questions | Answer Generation
+`task841_kde4_translation` | Localizing English phrases to Hindi language | Language Translation
```
Comment (Palipoor) [inline tasks/README.md:10]: Please change "Language Translation" to "Translation"

File: tasks/README.md [hunk 2]
```diff
@@ -6,4 +6,9 @@ This directory contains the tasks that are part of this benchmark.
...
+`task842_kde4_translation` | Localizing English phrases to Telugu language | Language Translation
```
Comment (Palipoor) [inline tasks/README.md:11]: Also here.

File: tasks/README.md [hunk 3]
```diff
@@ -6,4 +6,9 @@ This directory contains the tasks that are part of this benchmark.
...
+`task843_schema_guided_dstc8_classification` | Classifying sentence as question or not | Classification
+`task844_schema_guided_dstc8_classification` | Classifying sentence into one of five action categories | Multi-Class Classification
```
Comment (Palipoor) [inline tasks/README.md:13]: "Classification" is enough.

File: tasks/README.md [hunk 4]
```diff
@@ -6,4 +6,9 @@ This directory contains the tasks that are part of this benchmark.
...
+`task845_schema_guided_dstc8_classification` | Identifying which of five services the sentence intent belongs to  | Multi-Class Classification
```
Comment (Palipoor) [inline tasks/README.md:14]: Also here.

**Comment by Palipoor** [comment]
The sentence `We are given a kde4 dataset, a parallel corpus of KDE4 localization files. ` is not needed in the tasks 841 and 842.
Please use second person pronouns in your descriptions, as if you're explaining the task to a friend.
Like:
`    "Definition": "We are given a schema-guided dialogue dataset that has annotated multi-domain, task-oriented conversations between a human and a virtual assistant. The task is to classify whether a given sentence is a question or not.",` 
=>
`     "Definition": "You are given a sentence from a conversation between a human and a virtual assistant. Your task is to classify whether a given sentence is a question or not. ",`
Please apply this change to all the descriptions.
Also, please change the classification labels to text instead of numbers(like you've done in task 844)

**Comment by RajaKashyap** [comment]
@Palipoor I have updated the files with suggested changes. Please verify.

**Comment by Palipoor** [comment]
They look good to me! Thanks.

**Comment by swarooprm** [comment]
@kashyap467, Just a reminder that, if you plan to submit these tasks to the main Github repo and join our research project, Monday (27th Sept) is the deadline. Again, this is optional and independent of your HW2 grade.

**Comment by kashyap467** [comment]
Hi Swaroop,
I have submitted my tasks to the repo 'allenai/natural-instructions-expansion'
with task numbers 645-649.

Regards,
Kashyap.

**Pull request closed without merging**